modules/installation-configuration-parameters.adoc

```diff
@@ -388,6 +388,10 @@ control plane machine pool.
 belong to the same region as the cluster. This is required for regions that require a custom {op-system} AMI.
 |Any published or custom {op-system} AMI that belongs to the set AWS region.
 
+|`platform.aws.hostedZone`
+|An existing Route 53 private hosted zone for the cluster. You can only use a pre-existing hosted zone when also supplying your own VPC. The hosted zone must already be associated with the user-provided VPC before installation. If undefined, the installation program creates a new hosted zone.
+|String, for example `Z3URY6TWQ91KVV`.
```

Comment:
@staebler , `hostedZone` ID should correspond to `baseDomain` name， right?

Reply:
@staebler @yunjiang29 from the [AWS docs](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/hosted-zone-private-creating.html), it sounds like the domain name is inputted when creating the private hosted zone. Should we warn that the private hosted zone must be associated with the cluster base domain AND user-provided VPC?

Reply:
@codyhoag good suggestion

fyi, @staebler has wrote some restrictions in [PR](https://github.com/openshift/installer/pull/4772#issue-596174283 )